Scripts/Ticket_163723.ps1

```diff
@@ -0,0 +1,45 @@
+﻿
+param (
+    [string] $dbType,
+    [string] $username 
```

Comment:
Why do you need username?

Reply:
@5380harshjain said:

> My understanding here is that we've to map the DB types with LU names in the script and then remove semicolons according to database type and lu name. So can we create key-value {dbType: LU_Name} pair in the script which will contain database type as key and lu_name as value and then iterate over all the database types which will be coming from pipeline and then check it with key-value pair and removing semicolons accordingly from the respective line. Is it the right approach? Please let me know.

Yes that sounds like the correct approach.

And I want to see Log Messages!

Reply:
@robe070 I've made some changes, please review and let me know.